lib/Gitlab/Api/Environments.php

```diff
@@ -0,0 +1,43 @@
+<?php namespace Gitlab\Api;
+
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class Environments extends AbstractApi
+{
+    /**
+     * @param int $project_id
+     * @return mixed
+     */
+    public function all($project_id)
+    {
+        return $this->get($this->getProjectPath($project_id, 'environments'));
+    }
+
+    /**
+     * @param int $project_id
+     * @param array $parameters
+     * @return mixed
+     */
+    public function create($project_id, array $parameters = array())
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefined('name')
```

Comment:
According to the documentation this parameter is required. Could you use `setRequired` instead?

Reply:
Done.